src/Styleguide/Components/AuctionResultItem.tsx

```diff
@@ -0,0 +1,182 @@
+    <Flex flexDirection="column" justifyContent="space-between">
+      <div>
+        <ScaledArtworkImage
+          src={props.imageUrl}
+          width="auto"
+          height="30px"
+          mr={4}
+          ml={4}
+        />
+      </div>
+    </Flex>
+    <Flex flexDirection="column" justifyContent="space-between">
+      <div>
+        <Serif size="3t">
+          <Title>{props.title}, </Title>
+          {props.date}
+        </Serif>
+        <MetadataContainer>
+          <Serif size="3t">{props.dimensions.in}</Serif>
+          <Serif size="3t">{props.dimensions.cm}</Serif>
+        </MetadataContainer>
+        <br />
+        <Serif size="3t">{props.organization}</Serif>
+        <MetadataContainer>
+          <Serif size="3t">{props.auctionDate}</Serif>
+        </MetadataContainer>
+        <Serif size="3t">Sale: {props.salePrice}</Serif>
+        <MetadataContainer>
+          <Serif size="3t">Est: {props.estimate}</Serif>
+        </MetadataContainer>
+      </div>
+    </Flex>
+  </Flex>
+)
+
+export const SmallAuctionResultItem = (props: AuctionResultItemProps) => (
+  <Flex p={3} width="100%" justifyContent="space-between">
+    <Flex flexDirection="column" justifyContent="space-between">
+      <div>
+        <ScaledArtworkImage
+          src={props.imageUrl}
+          width="auto"
+          height="70px"
+          mr={4}
+          ml={4}
+        />
+      </div>
+    </Flex>
+    <Flex flexDirection="column" justifyContent="space-between">
+      <div>
+        <Serif size="3t">
+          <Title>{props.title}, </Title>
+          {props.date}
+        </Serif>
+        <MetadataContainer>
+          <Serif size="3t">{props.dimensions.in}</Serif>
+          <Serif size="3t">{props.dimensions.cm}</Serif>
+        </MetadataContainer>
+        <br />
+        <Serif size="3t">{props.organization}</Serif>
+        <MetadataContainer>
+          <Serif size="3t">{props.auctionDate}</Serif>
+          <Serif size="3t">
+            <TextLink underline>View Details</TextLink>
+          </Serif>
+        </MetadataContainer>
+      </div>
+    </Flex>
+    <Flex flexDirection="column" justifyContent="space-between">
+      <div>
+        <Serif size="3t">Sale: {props.salePrice}</Serif>
+        <MetadataContainer>
+          <Serif size="3t">Est: {props.estimate}</Serif>
+        </MetadataContainer>
+      </div>
+    </Flex>
+  </Flex>
+)
+
+export const LargeAuctionResultItem = (props: AuctionResultItemProps) => (
+  <Flex p={4} width="100%" justifyContent="space-between">
+    <Flex flexDirection="column" justifyContent="space-between">
+      <div>
+        <ScaledArtworkImage
+          src={props.imageUrl}
+          width="auto"
+          height="70px"
+          mr={4}
+          ml={4}
+        />
+      </div>
+    </Flex>
+    <Flex flexDirection="column" justifyContent="space-between">
+      <div>
+        <Serif size="3t">
+          <Title>{props.title}, </Title>
```

Comment:
Should this be `<Serif size='3t' italic>{props.title}</Serif>`?

Reply:
Well, `Title` was using a `span` for italics to get the title, date on one line. Versus separate `Serif` tags for each, both are divs/blocks.

Reply:
Ah, if you want you can add `display='whatever'`